infrastructure/lib/kit-infrastructure.ts

```diff
@@ -21,7 +22,7 @@ export class KITInfrastructure extends Stack {
     const installEBSCSIDriverAddon = this.getContextOrDefault("AWSEBSCSIDriverAddon", "true")
     const installKarpenterAddon = this.getContextOrDefault('KarpenterAddon', "true")
     const installKitAddon = this.getContextOrDefault("KITAddon", "true")
-    const repoAddonPaths = this.node.tryGetContext('FluxRepoAddonPaths')
+    const installPerfDashAddon = this.getContextOrDefault("PerfDashAddon", "false")
```

Comment:
how do we install other addons if you intend to replace this to just perfdash ?

Reply:
`FluxRepoAddonPaths` was introduced for perfdash and removing it won't impact other addons

Reply:
I think it was introduced for not just perfdash but also for addons in this folder - https://github.com/awslabs/kubernetes-iteration-toolkit/tree/main/infrastructure/k8s-config/clusters/kit-infrastructure